src/meta/src/hummock/manager/mod.rs

```diff
@@ -921,7 +921,7 @@ impl HummockManager {
         let developer_config = Arc::new(CompactionDeveloperConfig::new_from_meta_opts(
             &self.env.opts,
         ));
-        const MAX_TRIVIAL_MOVE_TASK_COUNT: usize = 256;
+        const MAX_TRIVIAL_MOVE_TASK_COUNT: usize = 64;
```

Comment:
I do not suggest adjust this variable until we really meet problem about it, otherwise we may make meta-node more busy after upgrading.

Reply:
Revert it and add new config `max_trivial_move_task_count_per_loop`